packages/twenty-front/src/modules/ui/layout/dropdown/components/DropdownMenuSearchInput.tsx

```diff
@@ -35,12 +36,15 @@ const StyledInput = styled.input`
 export const DropdownMenuSearchInput = forwardRef<
   HTMLInputElement,
   InputHTMLAttributes<HTMLInputElement>
->(({ value, onChange, autoFocus, placeholder = 'Search', type }, ref) => (
-  <StyledDropdownMenuSearchInputContainer>
-    <StyledInput
-      autoComplete="off"
-      {...{ autoFocus, onChange, placeholder, type, value }}
-      ref={ref}
-    />
-  </StyledDropdownMenuSearchInputContainer>
-));
+>(({ value, onChange, placeholder = 'Search', type }) => {
+  const {inputRef} = useFocus()
+  return (
+    <StyledDropdownMenuSearchInputContainer>
+      <StyledInput
```

Comment:
Could you try to work on the StyledInput component level ? Ideal solution would be that this useEffect is not an external hook but rather something internal to the input component and we can pass "preventScroll" as a property of this input.

It could be fine to create a new input component if you feel the need.